galsim/chromatic.py

```diff
@@ -162,12 +381,28 @@ def drawImage(self, bandpass, image=None, integrator=None, **kwargs):
                                 integrate.
         @param image            Optionally, the Image to draw onto.  (See GSObject.drawImage()
                                 for details.)  [default: None]
-        @param integrator       One of the image integrators from galsim.integ [default: None,
+        @param integrator       When doing the exact evaluation of the profile, this argument should
+                                be one of the image integrators from galsim.integ [default: None,
                                 which will try to select an appropriate integrator automatically.]
+                                If setupInterpolation() has been called for this object, then
+                                `integrator` should be a string, either 'midpoint' or
+                                'trapezoidal'. [default: 'trapezoidal']
```

Comment:
But for the normal integration method, I think it�s more complicated than this.  Users have to choose one of the integration methods like sample integrator or continuous integrator, and  either the midpoint or trapezoidal method.  So I don�t see that using just a string for �midpoint� or �trapezoidal� will do the job.